src/v/raft/replicate_entries_stm.cc

```diff
@@ -151,8 +151,12 @@ inline bool replicate_entries_stm::is_follower_recovering(model::node_id id) {
 ss::future<result<replicate_result>>
 replicate_entries_stm::apply(ss::semaphore_units<> u) {
     // first append lo leader log, no flushing
+    auto cfg = _ptr->config();
```

Comment:
do you need a copy?

Reply:
yes, I need to use the same configuration for the whole continuation chain